src/ngraph/op/softmax.hpp

```diff
@@ -53,5 +53,47 @@ namespace ngraph
         private:
             AxisSet m_axes;
         };
+
+        namespace set0
+        {
+            using ngraph::op::Softmax;
+        }
+
+        namespace set1
+        {
+            class Softmax : public util::UnaryElementwiseArithmetic
+            {
+            public:
+                NGRAPH_API
+                static const std::string type_name;
+                const std::string& description() const override { return type_name; }
+                Softmax()
+                    : m_axis(0)
+                {
+                    set_opset_version(1);
+                }
+                /// \brief Constructs a softmax operation.
+                ///
+                /// \param arg Node that produces the first input tensor.<br>
+                /// `[d0, ...]`
+                /// \param axis The axis positions (0-based) on which to calculate the softmax.
```

Comment:
`The axis position` instead of `The axis positions` sounds better IMO.
But I won't force this change.

Reply:
Done